Base/usr/share/man/man5/Shell.md

```diff
@@ -211,9 +211,9 @@ if A {
 ##### For Loops
 For Loops evaluate a sequence of commands once per element in a given list.
 The shell has two forms of _for loops_, one with an explicitly named iteration variable, and one with an implicitly named one.
-The general syntax follows the form `for name in expr { sequence }`, and allows omitting the `name in` part to implicitly name the variable `it`.
+The general syntax follows the form `for enum enum_name name in expr { sequence }`, and allows omitting the `enum enum_name name in` part to implicitly name the variable `it`.
```

Comment:
enum_name? I can't help but think about c++ enums rather than enumerate. Python docs say that it returns an object such that "the __next__() method of the iterator returned by enumerate() returns a tuple containing a count (from start which defaults to 0) and the values obtained from iterating over iterable.". So, that would make "enum_name" the "count" right? Or "index" is how I normally see it used. `for idx, foo in enumerate(foos)`.

Reply:
`index_name` would be more appropriate I guess.

Reply:
Renamed to `for index`, and changed the manpage too.